biggus/tests/unit/test__dual_input_fn_wrapper.py

```diff
@@ -0,0 +1,102 @@
+        wrapped_fn = biggus._dual_input_fn_wrapper('my_module.my_function',
+                                                   lambda a, b: a + b)
+        self.assertEqual(wrapped_fn.__name__, (lambda a, b: a + b).__name__)
+
+    def test_given_fn_name(self):
+        wrapped_fn = biggus._dual_input_fn_wrapper('my_module.my_function',
+                                                   lambda a, b: a + b,
+                                                   fn_name='identity')
+        self.assertEqual(wrapped_fn.__name__, 'identity')
+
+    def test_masked_array_creates_elementwise(self):
+        wrapped_fn = biggus._dual_input_fn_wrapper('my_module.my_function',
+                                                   lambda a, b: a + b,
+                                                   lambda a, b: a - b,
+                                                   fn_name='identity')
+        result = wrapped_fn(np.array([-5, 2]), np.array([-5, 2]))
+        self.assertIsInstance(result, biggus._Elementwise)
+
+    def test_ndarray_expected_values(self):
+        wrapped_fn = biggus._dual_input_fn_wrapper('my_module.my_function',
+                                                   lambda a, b: a + b,
+                                                   fn_name='identity')
+        result = wrapped_fn(np.array([-5, 2]), np.array([-5, 2]))
+        assert_array_equal(result.ndarray(), [-10, 4])
+
+    def test_masked_array_expected_values(self):
+        wrapped_fn = biggus._dual_input_fn_wrapper('my_module.my_function',
+                                                   lambda a, b: a - b,
+                                                   lambda a, b: a + b,
+                                                   fn_name='identity')
+        result = wrapped_fn(np.array([-5, 2]), np.array([-5, 2]))
+        assert_array_equal(result.masked_array(), [-10, 4])
+
+    def test_masked_array_undefined(self):
+        wrapped_fn = biggus._dual_input_fn_wrapper('my_module.my_function',
+                                                   lambda a, b: a + b,
+                                                   fn_name='identity')
+        msg = 'No <lambda> operation defined for masked arrays.'
+        result = wrapped_fn(np.array([-5, 2]), np.array([-5, 2]))
+        with self.assertRaisesRegexp(TypeError, msg):
+            assert_array_equal(result.masked_array(), [0, 4])
+
+
+class Test_wrapped_functions(unittest.TestCase):
+    def test_all_fns(self):
+        fns_to_test = ['copysign', 'nextafter', 'ldexp', 'fmod']
+        arr1 = np.array([10, 2, 5])
+        arr2 = np.array([1, 5, 15])
+        biggus_arr1 = biggus.NumpyArrayAdapter(arr1)
+        biggus_arr2 = biggus.NumpyArrayAdapter(arr2)
+
+        for fn_name in fns_to_test:
+            np_fn = getattr(np, fn_name)
+            biggus_fn = getattr(biggus, fn_name)
+            result = biggus_fn(biggus_arr1, biggus_arr2)
+            assert_array_equal(result.ndarray(), np_fn(arr1, arr2))
```

Comment:
I dislike `for` loops inside test methods because they can 'hide' failing tests behind other failing tests that are executed earlier and so not show the full range of test failures in a single test run.

You _could_ split this up so that you have a common method that executes the test steps and one method for each of your `fns_to_test`... but honestly it's not a big deal and I'm not too bothered if you don't.